org.restlet.java/org.restlet/src/main/java/org/restlet/engine/ssl/DefaultSslContextFactory.java

```diff
@@ -675,6 +641,30 @@ public boolean isWantClientAuthentication() {
 		return wantClientAuthentication;
 	}
 
+	/**
+	 * Loads a keystore according to its file path, type and password.
+	 * @param path The file path of the keystore.
+	 * @param provider The name of the keystore provider.
+	 * @param type The keystore type of the keystore.
+	 * @param password the optional password of the keystore.
+	 * @return a keystore.
+	 * @throws Exception
+	 */
+	private static KeyStore loadKeyStore(String path, String provider, String type, char[] password) throws Exception {
```

Comment:
Could this method be non static and protected to allow overridding?

Reply:
sure, thanks